templates/company-board/README.md

```diff
@@ -0,0 +1,51 @@
+# Aragon Company Board template
+
+## Usage
+
+```
+company.prepareInstance(id)
+company.setupInstance(boardHolders, shareHolders, shareStakes)
```

Comment:
Shouldn't this be `newTokensAndInstance` instead? Where is `newTokens` called otherwise?

Reply:
`newTokens` will be merged into `prepareInstance`